backend/kesaseteli/applications/migrations/0034_make_social_security_number_optional.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.23 on 2024-04-16 13:46
+
+import common.utils
+from django.db import migrations
+import encrypted_fields.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('applications', '0033_add_non_vtj_birthdate_and_home_municipality'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='youthapplication',
+            name='social_security_number',
+            field=encrypted_fields.fields.SearchField(blank=True, db_index=True, encrypted_field_name='encrypted_social_security_number', hash_key='ee235e39ebc238035a6264c063dd829d4b6d2270604b57ee1f463e676ec44669', max_length=66, null=True, validators=[common.utils.validate_optional_finnish_social_security_number]),
```

Comment:
Is it okay to have the hash_key visible here?

Reply:
Searched and found issues in [django-searchable-encrypted-fields](https://gitlab.com/guywillett/django-searchable-encrypted-fields/-/tree/v0.1.9?ref_type=tags):
- https://gitlab.com/guywillett/django-searchable-encrypted-fields/-/issues/10
- https://gitlab.com/guywillett/django-searchable-encrypted-fields/-/issues/16
- https://gitlab.com/guywillett/django-searchable-encrypted-fields/-/issues/18

It seems this is a known problem and has been fixed in v0.2.1. Kesäseteli is currently using v0.1.9 (see requirements.txt). Created ticket [YJDH-700](https://helsinkisolutionoffice.atlassian.net/browse/YJDH-700) to our backlog for this issue.